docs/INSTALL-devel-flowchart.txt

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
not changing any HaaS to HIL.
We decided that to be a dedicated distinct PR. Doing just that.